.gitignore

```diff
@@ -1 +1,4 @@
 /vendor/
+/build/
+/Tests/Fixtures/project/var/cache/*
+/Tests/Fixtures/project/var/logs/*
```

Comment:
Good rule is to add new newline at the end of each file. For PHP files it's a rule http://www.php-fig.org/psr/psr-2/#22-files, but for any other files would be nice to have too.